### PR TITLE
Branch v1.0

### PR DESCRIPTION
**Pull request by bdthanh**
Please check everything, then we will create  jar file for v1.0 tmr tuesday

### CODE REVIEW

**Comment by bdthanh** [comment]
if ok merge and pull to your local and test before release v1.0

**Review by jingwei55 — COMMENTED** [review]
Code looks good! Should be able to make Jar file for our version 1. Later on can add more exception handling

File: src/main/java/seedu/duke/Ui.java [hunk 1]
```diff
@@ -55,47 +57,47 @@ public static String readInput() {
...
-    
+
     public static void addTransactionMessage(Transaction transaction, int listSize) {
         showLine();
         assert listSize >= 0;
-        System.out.println("OK! I will add the following transaction:\n" + transaction.toString() + "\n"
+        System.out.print("OK! I will add the following transaction:\n" + transaction.toString() + "\n"
                 + "Total transaction(s) in list: " + listSize + '\n');
         showLine();
     }
```
[the remark this answers is not in the document]
Comment (bdthanh) [reply src/main/java/seedu/duke/Ui.java]: Thanks for pointing it out. Fixed

File: src/main/java/seedu/duke/command/ListCommandsCommand.java [hunk 2]
```diff
@@ -1,5 +1,7 @@
 package seedu.duke.command;
 
+import seedu.duke.Ui;
+
 import java.util.Map;
 
 public class ListCommandsCommand extends Command {
```
Comment (jorellesee) [inline src/main/java/seedu/duke/command/ListCommandsCommand.java:7]: can consider renaming this function to avoid confusion with ListCommand

**Comment by chiewyx** [comment]
code looks good to me, can go ahead and merge

**Review by winston-lim — APPROVED** [review]
LGTM i think no changes further are required for first iteration

Also side note: Not related to this PR, but all of our changes at some point where lost because someone overwrote it unintentionally with their merge commit and tbh this is not workable with(especially since we're ignoring the forking workflow and largely not doing PR reviews)

TLDR: Can we all ensure to pull the latest code right before merging any PR with `git pull upstream master` **even if it shows no conflicts**
- Pulling at the start is not sufficient - do it right before a merge(clicking on merge PR) as well

**Review by chiewyx — APPROVED** [review]
good to merge

File: src/main/java/seedu/duke/command/ListCommandsCommand.java (hunk 2 above)
Comment (bdthanh) [reply src/main/java/seedu/duke/command/ListCommandsCommand.java]: i changed to HelpCommand
